javascript/index.html

```diff
@@ -111,6 +111,10 @@ <h4>
                 <a href="./answers/Dhanu084-q5.html">
                     <h4>What are escape Sequences?</h4>
                 </a>
+            </li>
+            <li>
+                <a href="./answers/ayushk12-q1.html"><h4>What is the value of !'bang'?</h4></a>
```

Comment:
It's not q1, take a fresh pull and update the q(no.)

Reply:
updated